lib/feature/home/presentation/widgets/photo_tab/button_tabs.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:file_share/core/presentation/palette.dart';
+import 'package:flutter/material.dart';
+
+class ButtonTabs extends StatefulWidget {
+  final int photoCount;
+  const ButtonTabs({
+    super.key,
+    required this.photoCount,
+  });
+
+  @override
+  State<ButtonTabs> createState() => _ButtonTabsState();
+}
+
+class _ButtonTabsState extends State<ButtonTabs> {
+  bool imagesButton = true;
+  bool albumsButton = false;
+  int selectedButtonIndex = 0;
+
+  void toggle(int index) {
+    setState(() {
```

Comment:
This setState here is unnecessary because you're still calling setState at the end of this method.
Just set `selectedButtonIndex = index;` and the setState at the bottom of the method will notify the UI to rebuild just once.

What you have here will cause two rebuilds (one being unnecessary).